archetypes/quickstart/src/brooklyn-sample/pom.xml

```diff
@@ -58,19 +53,30 @@
       <releases>
         <enabled>false</enabled>
       </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
     </repository>
   </repositories>
 
   <dependencies>
     <dependency>
-      <!-- this pulls in all brooklyn entities and clouds; 
-           you can cherry pick selected ones instead (for a smaller build) -->
+      <!-- minimal dependency, for writing entities; consider also brooklyn-software-base -->
```

Comment:
May be list available software modules and what they provide (ex. web, nosql, database, messaging, etc...). How does one figure out which one to add, given a class name (say Tomcat8Server)?